src/main/java/step3/controller/RacingCarController.java

```diff
@@ -1,22 +1,27 @@
 package step3.controller;
 
 import step3.model.Cars;
+import step3.model.MoveStrategy;
+import step3.model.RaceWinners;
 import step3.view.InputView;
 import step3.view.OutputView;
 
 public class RacingCarController {
+
     public RacingCarController() {
     }
 
     public void racing() {
-        int carCount = InputView.getCarCount();
+        Cars cars = Cars.from(InputView.getCarNames());
         int tryCount = InputView.getTryCount();
-
-        Cars cars = Cars.from(carCount);
         OutputView.printRacingResult();
+
         for (int i = 0; i < tryCount; i++) {
-            cars.moveCars();
+            cars.moveCars(new MoveStrategy());
             OutputView.printRacingProgress(cars);
         }
+
+        RaceWinners raceWinners = new RaceWinners(cars.getCars());
```

Comment:
경주가 완료된 시점의 자동차들이 `Cars`에게 있으니, 우승자를 구하는 책임을 `Cars`에게 부여해보는 건 어떨까요?
```suggestion
        RaceWinners raceWinners = cars.getRaceWinners();
```

Reply:
넵 괜찮습니다 혹시 `Cars`에서 `RaceWinners`를 생성한다는 부분에서 고민되는 부분이 있으실까요?

Reply:
제가 이해를 잘못한 것 같습니다.. 'Cars' 에서 'RaceWinners'를 생성하는다는 것이 과연 'Cars'에서 로직을 구현하는것이 "올바른 책임인가?" 라는 고민을 했었어요

Reply:
제가 이해가 잘 안 되는데 '`Cars`에서 로직을 구현하는 것'이 어떤 로직을 의미하는지 여쭤봐도 될까요?